tasks/post_processing.wdl

```diff
@@ -0,0 +1,67 @@
+version development
+
+''' 
+Tasks to aggregate results from multiple scatters 
+(chromosome-gene pairs) each running CellRegMap functions
+'''
+
+# takes as input one file per scatter, returns both 
+# all results and all significant results (at a given FDR)
+
+task AggregateInteractionResults { # results from RunInteraction
+    input {
+        Array[File] listOfFiles 
+        Float FDR_threshold # false discovery rate
```

Comment:
Aim for consistent casing, most variables use camelCase, suggesting `fdrThreshold` or even `falseDiscoveryRateThreshold`:

```suggestion
        Float fdrThreshold # false discovery rate
```